README.rst

```diff
@@ -1,76 +1,61 @@
+* `Astropy community engagement study <https://zenodo.org/records/10603049>`_
+* `Astropy diversity, equity and inclusion study <https://astropy-dei.orgmycology.com/>`_
 
 Installation
 ============
 
-Releases are `registered on PyPI <https://pypi.org/project/astropy>`_,
-and development is occurring at the
-`project's GitHub page <https://github.com/astropy/astropy>`_.
-
-For detailed installation instructions, see the
-`online documentation <https://docs.astropy.org/>`_
-or `docs/install.rst <docs/install.rst>`_ in this source distribution.
-
 To install `astropy` from PyPI, use:
 
 .. code-block:: bash
 
     pip install astropy
 
+For more detailed instructions, see the `install guide <https://docs.astropy.org/en/stable/install.html>`_ in the docs.
+
 Contributing
 ============
 
 The Astropy Project is made both by and for its users, so we welcome and
 encourage contributions of many kinds. Our goal is to keep this a positive,
-inclusive, successful, and growing community by abiding with the
+inclusive, successful, and growing community that abides by the
 `Astropy Community Code of Conduct <https://www.astropy.org/about.html#codeofconduct>`_.
 
-More detailed information on contributing to the project or submitting feedback
-can be found on the `contributions page <https://www.astropy.org/contribute.html>`_.
-A `summary of contribution guidelines <CONTRIBUTING.md>`_ can also be
-used as a quick reference when you are ready to start writing or validating
-code for submission.
-
-Getting Started with GitHub Codespaces
-======================================
+For guidance on contributing to or submitting feedback for the Astropy Project, see the
+`contributions page <https://www.astropy.org/contribute.html>`_.
+For contributing code specifically, the developer docs have a `guide <https://docs.astropy.org/en/latest/index_dev.html>`_ with a quickstart.
+There's also a `summary of contribution guidelines <CONTRIBUTING.md>`_.
 
-Codespaces is a cloud development environment supported by GitHub.
-None of the Astropy build machinery depends on it, but it is a
-convenient way to quickly get started doing development on Astropy.
+Developing with Codespaces
+==========================
 
-To get started, create a codespace for this repository by clicking this:
+GitHub Codespaces is a cloud development environment using Visual Studio Code in your browser.
+This is a convenient way to start developing Astropy, using our `dev container <.devcontainer/devcontainer.json>`_ configured
+with the required packages. For help, see the `GitHub Codespaces
+docs <https://docs.github.com/en/codespaces>`_.
 
 |Codespaces|
 
-A codespace will open in a web-based version of Visual Studio Code.
-The `dev container <.devcontainer/devcontainer.json>`_ is fully configured
-with software needed for this project. For help, see the `GitHub Codespaces
-Support page <https://docs.github.com/en/codespaces>`_.
-
-**Note**: Dev containers is an open spec which is supported by
```

Comment:
Turns out the original Codespaces blurb, including the note, was originally added by GitHub Codespaces developer in #14481 . Do we really want to reword the section added by Codespaces expert? Is there a strong reason to do so?

Reply:
The previous text was kind of long and had some legal speak. I think it makes sense in a repo about codespaces, but a big disclaimer isn't needed in every repo where codespaces is used. I think the README should be as short as possible to be useful as a quick reference, with longer comments on things like codespaces better left to a page in the docs if it's needed.

Reply:
I think the reword is clearer!